sites/skeleton.dev/src/routes/(inner)/utilities/lightswitches/+page.svelte

```diff
@@ -60,21 +61,47 @@
 							operate. It is best suited within a fixed header element that's always visible on the page.
 						</p>
 					{:else if tabSection === 1}
-						<!-- OS Preference -->
-						<p>
-							Alternatively, Skeleton provides a utility method to adjust the mode based on operating system preference. Use this in place
-							of a Lightswitch component.
-						</p>
-						<p>First, import the following in <code class="code">/src/routes/+layout.svelte</code>.</p>
-						<CodeBlock language="ts" code={`import { autoModeWatcher } from '@skeletonlabs/skeleton';`} />
-						<p>Then add the following in your root layout template markup.</p>
-						<CodeBlock language="html" code={snippetAutoModeWatcher} />
-						<p>Note that Mac OS will update immediately, while other operating systems require a browser refresh.</p>
+						<TabGroup regionPanel="space-y-4">
+							<Tab bind:group={osTabs} name="interface" value={0}>SSR</Tab>
+							<Tab bind:group={osTabs} name="interface" value={1}>No SSR</Tab>
+							<svelte:fragment slot="panel">
+								{#if osTabs === 0}
+									<!-- OS Preference -->
+									<h2 class="h2">SSR</h2>
+									<p>
+										Skeleton provides a utility method to adjust the mode based on operating system preference. Use this in place of a
+										Lightswitch component.
+									</p>
+									<p>First, import the following in <code class="code">/src/routes/+layout.svelte</code>.</p>
+									<CodeBlock language="ts" code={`import { autoModeWatcher } from '@skeletonlabs/skeleton';`} />
+									<p>Then add the following in your root layout template markup.</p>
+									<CodeBlock language="html" code={snippetAutoModeWatcher} />
+
+									<p>Note that Mac OS will update immediately, while other operating systems require a browser refresh.</p>
+								{:else}
+									<h2 class="h2">No SSR</h2>
+									<p>Your <code class="code">/src/routes/+layout.ts</code> should look like this</p>
+									<CodeBlock language="ts" code={`export const ssr = false;`} />
+									<p>Add the following to your root <code class="code">/src/routes/+layout.svelte</code> file</p>
+									<CodeBlock
+										language="ts"
+										code={`
+import { autoModeWatcher } from '@skeletonlabs/skeleton'; 
+import { onMount } from 'svelte';
+
+onMount(() => {					
+	autoModeWatcher.toString();						
```

Comment:
I can't tell what you are implying. onMount is required to be called so that `window, document` both exist at call time, otherwise `autoModeWatcher` does not work, and there is no change of styles from light-dark.
If you are referring to the extra `autoModeWatcher.toString()`, that will be removed.

Reply:
I'm referring exactly on this line only. `toString()` return a string and assigned to nothing. I should be more clear. So this is noop. 

It was for SvelteKit because there's no import in the layout and simply "print" the code of the function definition to be called. I can tell because I just noticed this nice hack in the original layout then found this PR.

Reply:
Understood